src/site/twitch.tv/modules/emote-menu/EmoteMenuTab.vue

```diff
@@ -90,6 +98,20 @@ const favorites = useConfig<Set<string>>("ui.emote_menu.favorites");
 // const usage = useConfig<Map<string, number>>("ui.emote_menu.usage");
 // const shouldShowUsage = useConfig<boolean>("ui.emote_menu.most_used");
 
+//Emote icons for groups
```

Comment:
nit: add whitespace at comment start

```suggestion
// Emote icons for groups
```

Reply:
Added whitespace in 1dda7d0